### PR TITLE
update format, wording, and UI text

### PR DESCRIPTION
**Pull request by Oreoxmt**

### CODE REVIEW

File: tidb-cloud/backup-and-restore.md [hunk 1]
```diff
@@ -15,101 +15,105 @@ This document describes how to back up and restore your TiDB Dedicated cluster d
...
 
-- If you turn on and off Point-in-time Restore (Point-in-time Restore) multiple times, you can only choose a time point within the recoverable range after the most recent Point-in-time Restore is enabled. The earlier recoverable range is not accessible.
+## Backup
 
-## Turn on auto backup
+### Configure automatic backup
 
-TiDB Dedicated supports both [snapshot backups](https://docs.pingcap.com/tidb/stable/br-snapshot-guide) and [log backups](https://docs.pingcap.com/tidb/stable/br-pitr-guide). Snapshot backup enables users to restore to the the backup point. By default, snapshot backups are taken automatically and stored according to your backup retention policy.
+TiDB Dedicated supports both [snapshot backup](https://docs.pingcap.com/tidb/stable/br-snapshot-guide) and [log backup](https://docs.pingcap.com/tidb/stable/br-pitr-guide) (Point-in-time Restore in TiDB Dedicated). Snapshot backup enables you to restore to a specific backup point. By default, TiDB Dedicated automatically performs snapshot backup and stores the backup files according to your backup retention policy.
```
Comment (qiancai) [inline tidb-cloud/backup-and-restore.md]:
```suggestion
TiDB Dedicated supports both [snapshot backup](https://docs.pingcap.com/tidb/stable/br-snapshot-guide) and [log backup](https://docs.pingcap.com/tidb/stable/br-pitr-guide) (Point-in-time Restore in TiDB Dedicated). Snapshot backup enables you to restore data to a specific backup point. By default, TiDB Dedicated automatically performs snapshot backup and stores backup files according to your backup retention policy.
```

Comment (qiancai) [reply tidb-cloud/backup-and-restore.md]: (Point-in-time Restore in TiDB Dedicated) 写到这里是什么意思呢

File: tidb-cloud/backup-and-restore.md [hunk 2]
```diff
@@ -15,101 +15,105 @@ This document describes how to back up and restore your TiDB Dedicated cluster d
...
-You can disable auto backup at any time.
+To configure automatic backup, perform the following steps:
 
-### Turn on point-in-time restore / log backup
+1. Navigate to the **Backup** page of a TiDB Dedicated cluster.
 
-> **Note**
->
-> Point-in-time Restore / Log Backup is supported from **TiDB v6.4.0**.
+2. Click **Backup Settings**. The setting window displays.
```
Comment (qiancai) [inline tidb-cloud/backup-and-restore.md]:
```suggestion
2. Click **Backup Settings**. The setting window is displayed.
```

File: tidb-cloud/backup-and-restore.md [hunk 3]
```diff
@@ -15,101 +15,105 @@ This document describes how to back up and restore your TiDB Dedicated cluster d
...
+In the **Backup Settings** window, you can turn on the feature with the following steps:
 
-To configure backup schedule:
+1. Toggle the **Auto Backup** switch to **On**.
+2. Toggle the **Point-in-time Restore** switch to **On**.
 
-1. Navigate to the **Backup** page of a TiDB Dedicated cluster.
+> **Warning**
+>
+> Point-in-Time Restore only takes effect after the next backup task completes. To enable it immediately, you need to [manually perform a backup](#manual-backup) after configuration.
```
Comment (qiancai) [inline tidb-cloud/backup-and-restore.md]:
```suggestion
> Point-in-Time Restore only takes effect after the next backup task is completed. To make it take effect earlier, you can [manually perform a backup](#manual-backup) after enabling it.
```

File: tidb-cloud/backup-and-restore.md [hunk 4]
```diff
@@ -15,101 +15,105 @@ This document describes how to back up and restore your TiDB Dedicated cluster d
...
 
-2. Click **Backup Settings**.
+#### Set up backup scheduler
 
-3. Under **Backup Schedule**, choose either the **Daily** or **Weekly** option. If you choose **Weekly**, specify the days of the week for the backup.
+TiDB Dedicated supports daily and weekly backup schedules. By default, the backup schedule is set to daily. You can choose a specific time of the day or week to start snapshot backup.
 
-4. In **Backup Time**, set a start time for the daily or weekly cluster backup. By default, it's a daily backup starting from 2:00 AM in the time zone of the cluster's region.
+To configure backup schedule, toggle the **Auto Backup** switch to **On** and set the following settings:
```
Comment (qiancai) [inline tidb-cloud/backup-and-restore.md]:
```suggestion
To configure the backup schedule, toggle the **Auto Backup** switch to **On** and make the following settings:
```

File: tidb-cloud/backup-and-restore.md [hunk 5]
```diff
@@ -15,101 +15,105 @@ This document describes how to back up and restore your TiDB Dedicated cluster d
...
 
-     > **Note**
-     >
-     > - Scheduling automatic backup jobs during periods of low workload is recommended to minimize the impact on business operations.
-     > - Backup jobs will be automatically delayed when importing data jobs are in progress. Do not run the manual backup during data import jobs or cluster scaling.
+- In **Backup Scheduler**, select either the **Daily** or **Weekly** checkbox. If you select **Weekly**, you need to specify the days of the week for the backup.
 
+    > **Warning**
+    >
+    > - The Point-in-time Restore feature is enabled by default and cannot be disabled when you enable weekly backup.
```
Comment (qiancai) [inline tidb-cloud/backup-and-restore.md]:
```suggestion
    > - When weekly backup is enabled, the Point-in-time Restore feature is enabled by default and cannot be disabled.
```

File: tidb-cloud/backup-and-restore.md [hunk 6]
```diff
@@ -15,101 +15,105 @@ This document describes how to back up and restore your TiDB Dedicated cluster d
...
-5. In **Backup Retention**, configure the minimum backup data retention period. The default is 7 days.
+- In **Backup Time**, schedule a start time for the daily or weekly cluster backup.
+
+    If you do not specify a preferred backup time, TiDB Cloud assigns a default backup time, which is 2:00 AM in the time zone of the region where the cluster is located.
 
     > **Note**
     >
-    > - After you delete the cluster, the automatic backup files will be kept for a specific duration, as set in Backup Settings. Remember to delete these backup files as needed.
-    > - After you delete a cluster, the existing manual backup files will be retained until you manually delete them, or your account is closed.
+    > - To minimize the impact on business, it is recommended to schedule automatic backup at a low workload period.
```
Comment (qiancai) [inline tidb-cloud/backup-and-restore.md]:
```suggestion
    > - To minimize the impact on business, it is recommended to schedule automatic backup during periods of low workloads.
```

File: tidb-cloud/backup-and-restore.md [hunk 7]
```diff
@@ -15,101 +15,105 @@ This document describes how to back up and restore your TiDB Dedicated cluster d
...
+    > - Backup jobs are automatically delayed when importing data jobs are in progress. **DO NOT** run the manual backup while importing data, or during cluster scaling.
```
Comment (qiancai) [inline tidb-cloud/backup-and-restore.md]:
```suggestion
    > - Backup jobs are automatically delayed when data import jobs are in progress. **DO NOT** run manual backups during data import, or during cluster scaling.
```

File: tidb-cloud/backup-and-restore.md [hunk 8]
```diff
@@ -15,101 +15,105 @@ This document describes how to back up and restore your TiDB Dedicated cluster d
...
+- In **Backup Retention**, configure the minimum backup data retention period. The default is seven days.
 
-7. Click **Confirm** again to save changes.
-
-## Turn on dual region backup
-
-TiDB Dedicated supports dual region backup by copying backups to the new region as selected. After enabling it, all backups will be automatically copied to the specified region, providing an cross region data protection and disaster recovery capabilities. Approximately 99% of the data could be copied to the additional region within 1 hour.
+    > **Note**
+    >
+    > - After you delete the cluster, the automatic backup files will be retained for a specified period, as set in **Backup Settings**. You need to delete the backup files accordingly.
```
Comment (qiancai) [inline tidb-cloud/backup-and-restore.md:102]: as set in **Backup Retention**?

File: tidb-cloud/backup-and-restore.md [hunk 9]
```diff
@@ -15,101 +15,105 @@ This document describes how to back up and restore your TiDB Dedicated cluster d
...
 
     > **Note**
     >
-    > - After you delete the cluster, the automatic backup files will be kept for a specific duration, as set in Backup Settings. Remember to delete these backup files as needed.
-    > - After you delete a cluster, the existing manual backup files will be retained until you manually delete them, or your account is closed.
+    > - To minimize the impact on business, it is recommended to schedule automatic backup at a low workload period.
+    > - Backup jobs are automatically delayed when importing data jobs are in progress. **DO NOT** run the manual backup while importing data, or during cluster scaling.
 
-6. Click **Confirm** to preview the configuration change.
+- In **Backup Retention**, configure the minimum backup data retention period. The default is seven days.
```
Comment (qiancai) [inline tidb-cloud/backup-and-restore.md:93]: minimum?

File: tidb-cloud/backup-and-restore.md [hunk 10]
```diff
@@ -15,101 +15,105 @@ This document describes how to back up and restore your TiDB Dedicated cluster d
...
+    > **Note**
+    >
+    > - After you delete the cluster, the automatic backup files will be retained for a specified period, as set in **Backup Settings**. You need to delete the backup files accordingly.
+    > - After you delete a cluster, the existing manual backup files will be retained until you manually delete them, or your account is closed.
 
-Dual region backup costs include both backup storage usage and cross region data transfer fees. For more information, refer to: [Data Backup Cost](https://www.pingcap.com/tidb-dedicated-pricing-details/#backup-storage-cost).
+### Configure dual region backup
 
-To turn on dual region backup:
+TiDB Dedicated supports dual region backup by replicating backups to a region different from the cluster region. After you enable this feature, all backups are automatically replicated to the specified region. This provides cross-region data protection and disaster recovery capabilities. It is estimated that approximately 99% of the data can be replicated to the secondary region within an hour.
```
Comment (qiancai) [inline tidb-cloud/backup-and-restore.md]:
```suggestion
TiDB Dedicated supports dual region backup by replicating backups from your cluster region to another different region. After you enable this feature, all backups are automatically replicated to the specified region. This provides cross-region data protection and disaster recovery capabilities. It is estimated that approximately 99% of the data can be replicated to the secondary region within an hour.
```

File: tidb-cloud/backup-and-restore.md [hunk 11]
```diff
@@ -15,101 +15,105 @@ This document describes how to back up and restore your TiDB Dedicated cluster d
...
 
-1. Navigate to the **Backup** page of a TiDB Dedicated cluster.
+Dual region backup costs include both backup storage usage and cross region data transfer fees. For more information, refer to [Data Backup Cost](https://www.pingcap.com/tidb-dedicated-pricing-details#backup-storage-cost).
```
Comment (qiancai) [inline tidb-cloud/backup-and-restore.md]:
```suggestion
Dual region backup costs include both backup storage usage and cross-region data transfer fees. For more information, refer to [Data Backup Cost](https://www.pingcap.com/tidb-dedicated-pricing-details#backup-storage-cost).
```

File: tidb-cloud/backup-and-restore.md [hunk 12]
```diff
@@ -15,101 +15,105 @@ This document describes how to back up and restore your TiDB Dedicated cluster d
...
 
-- If you turn on and off Point-in-time Restore (Point-in-time Restore) multiple times, you can only choose a time point within the recoverable range after the most recent Point-in-time Restore is enabled. The earlier recoverable range is not accessible.
+## Backup
 
-## Turn on auto backup
+### Configure automatic backup
 
-TiDB Dedicated supports both [snapshot backups](https://docs.pingcap.com/tidb/stable/br-snapshot-guide) and [log backups](https://docs.pingcap.com/tidb/stable/br-pitr-guide). Snapshot backup enables users to restore to the the backup point. By default, snapshot backups are taken automatically and stored according to your backup retention policy.
+TiDB Dedicated supports both [snapshot backup](https://docs.pingcap.com/tidb/stable/br-snapshot-guide) and [log backup](https://docs.pingcap.com/tidb/stable/br-pitr-guide) (Point-in-time Restore in TiDB Dedicated). Snapshot backup enables you to restore data to a specific backup point. By default, TiDB Dedicated automatically performs snapshot backup and stores backup files according to your backup retention policy.
```
Comment (Oreoxmt) [inline tidb-cloud/backup-and-restore.md]:
```suggestion
TiDB Dedicated supports both [snapshot backup](https://docs.pingcap.com/tidb/stable/br-snapshot-guide) and [log backup](https://docs.pingcap.com/tidb/stable/br-pitr-guide). Snapshot backup enables you to restore data to a specific backup point. By default, TiDB Dedicated automatically performs snapshot backup and stores backup files according to your backup retention policy.
```

File: tidb-cloud/backup-and-restore.md [hunk 13]
```diff
@@ -15,101 +15,105 @@ This document describes how to back up and restore your TiDB Dedicated cluster d
...
-## Turn on dual region backup
-
-TiDB Dedicated supports dual region backup by copying backups to the new region as selected. After enabling it, all backups will be automatically copied to the specified region, providing an cross region data protection and disaster recovery capabilities. Approximately 99% of the data could be copied to the additional region within 1 hour.
+    > **Note**
+    >
+    > - After you delete the cluster, the automatic backup files will be retained for a specified period, as set in **Backup Settings**. You need to delete the backup files accordingly.
+    > - After you delete a cluster, the existing manual backup files will be retained until you manually delete them, or your account is closed.
 
-Dual region backup costs include both backup storage usage and cross region data transfer fees. For more information, refer to: [Data Backup Cost](https://www.pingcap.com/tidb-dedicated-pricing-details/#backup-storage-cost).
+### Configure dual region backup
```
Comment (Oreoxmt) [inline tidb-cloud/backup-and-restore.md]:
```suggestion
### Configure dual region backup

> **Note:**
>
> The dual region backup feature is currently in beta.
```

File: tidb-cloud/backup-and-restore.md [hunk 14]
```diff
@@ -15,101 +15,105 @@ This document describes how to back up and restore your TiDB Dedicated cluster d
...
 
-- If you turn on and off Point-in-time Restore (Point-in-time Restore) multiple times, you can only choose a time point within the recoverable range after the most recent Point-in-time Restore is enabled. The earlier recoverable range is not accessible.
+## Backup
 
-## Turn on auto backup
+### Configure automatic backup
 
-TiDB Dedicated supports both [snapshot backups](https://docs.pingcap.com/tidb/stable/br-snapshot-guide) and [log backups](https://docs.pingcap.com/tidb/stable/br-pitr-guide). Snapshot backup enables users to restore to the the backup point. By default, snapshot backups are taken automatically and stored according to your backup retention policy.
+TiDB Dedicated supports both [snapshot backup](https://docs.pingcap.com/tidb/stable/br-snapshot-guide) and [log backup](https://docs.pingcap.com/tidb/stable/br-pitr-guide). Snapshot backup enables you to restore data to a specific backup point. By default, TiDB Dedicated automatically performs snapshot backup and stores backup files according to your backup retention policy.
```
Comment (benmaoer) [inline tidb-cloud/backup-and-restore.md]:
```suggestion
TiDB Dedicated supports both [snapshot backup](https://docs.pingcap.com/tidb/stable/br-snapshot-guide) and [log backup](https://docs.pingcap.com/tidb/stable/br-pitr-guide). Snapshot backup enables you to restore data to the backup point. By default, snapshot backups are taken automatically and stored according to your backup retention policy.
```

Comment (benmaoer) [reply tidb-cloud/backup-and-restore.md]: Snapshot backup does not support to restore to a specific backup point

File: tidb-cloud/backup-and-restore.md [hunk 15]
```diff
@@ -15,101 +15,105 @@ This document describes how to back up and restore your TiDB Dedicated cluster d
...
+- In **Backup Retention**, configure the minimum backup data retention period. The default is seven days.
 
-7. Click **Confirm** again to save changes.
-
-## Turn on dual region backup
-
-TiDB Dedicated supports dual region backup by copying backups to the new region as selected. After enabling it, all backups will be automatically copied to the specified region, providing an cross region data protection and disaster recovery capabilities. Approximately 99% of the data could be copied to the additional region within 1 hour.
+    > **Note**
+    >
+    > - After you delete the cluster, the automatic backup files will be retained for a specified period, as set in **Backup Settings**. You need to delete the backup files accordingly.
```
Comment (Oreoxmt) [inline tidb-cloud/backup-and-restore.md:102]:
```suggestion
    > - After you delete the cluster, the automatic backup files will be retained for a specified period, as set in **Backup Retention**. You need to delete the backup files accordingly.
```

**Review by benmaoer — CHANGES_REQUESTED** [review]
Need to rebase by the latest pr

**Review by qiancai — APPROVED** [review]

**Pull request closed without merging**